pandas/tslib.pxd

```diff
@@ -1,7 +1,7 @@
 from numpy cimport ndarray, int64_t
 
 cdef convert_to_tsobject(object, object, object, bint, bint)
-cdef convert_to_timedelta64(object, object, object)
+cpdef convert_to_timedelta64(object, object)
```

Comment:
I had to, otherwise, it wouldn't be accessible in the Python world.   This is really inconsequential anyways if you read the docs <a href="http://notes-on-cython.readthedocs.io/en/latest/function_declarations.html">here</a>.